partcad/src/partcad/wrappers/wrapper_openscad.py

```
@@ -0,0 +1,80 @@
+        # If args is a list, join it as a comma-separated string
+        args_str = ", ".join(map(str, build_parameters))
+    elif isinstance(build_parameters, dict):
+        # If args is a dictionary, format it as key=value pairs
+        args_str = ", ".join(f"{k}={v}" for k, v in build_parameters.items())
+    else:
+        # Handle strings, numbers, or any other type by converting to string
+        args_str = str(build_parameters)
+
+    new_line = f"{method}({args_str});"
+
+    with open(tmp_file_path, "r") as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)